\documentclass[]{article}
\usepackage[round]{natbib}

\usepackage{fullpage}
\usepackage{listings}
\usepackage{url}
\usepackage{authblk}
\usepackage{graphicx}
\usepackage{color}
\usepackage{booktabs}

% lorem ipsum dummy text
\usepackage{lipsum}

\lstset{language=Python}

% cross-reference with main text
\usepackage{xr}
\externaldocument{paper}

% local definitions
\newcommand{\sgcomment}[1]{{\textcolor{red}{SG: #1}}}
\newcommand{\bmhcomment}[1]{{\textcolor{blue}{BMH: #1}}}
\newcommand{\aprcomment}[1]{{\textcolor{magenta}{APR: #1}}}
\newcommand{\tdwcomment}[1]{{\textcolor{cyan}{TDW: #1}}}

\begin{document}
\title{Supporting Information for ``A weakly structured stem for human origins in Africa''}
\author{Aaron P. Ragsdale, Timothy D. Weaver, Brenna M. Henn, and Simon Gravel}
\date{\today}
\maketitle

\renewcommand{\thefigure}{S\arabic{figure}}
\renewcommand{\thetable}{S\arabic{table}}
\renewcommand{\theequation}{S\arabic{equation}}
\setcounter{figure}{0}
\setcounter{table}{0}
\setcounter{equation}{0}

\tableofcontents
\newpage

\section{Data and sequencing}

\subsection{Sequencing and variant calling}
\cite{Kelleher2016-lw}

\subsection{Nama sample collection and consent}

\subsection{Details about populations used in analyses}

\subsection{Filtering and subsetting data}

\section{Computing statistics}

\subsection{Computing LD and diversity statistics}

\subsection{Estimating two-locus statistics with small sample sizes}

\subsection{Computing conditional SFS}

\section{Model specification and fitting}

\subsection{General strategy for building models and introducing complexity}

\subsection{Optimization using moments}

\subsection{Confidence intervals using godambe methods}

\section{Gene genealogy reconstruction}

\section{Predictions from inferred demographic models}

\subsection{$F_{ST}$ between coexisting populations over time}

\subsection{$f_4$ statistics between pairs of contemporary and pairs of ancient populations}

\section{Validations using simulatinos from inferred demographic models}

\subsection{Simulation details}

\subsection{cSFS prediction under inferred models}

\subsection{Relate curves from inferred models}

\subsection{Distribution of deep branch affinities to Neanderthal sequence}

\subsection{Mutation versus recombination rates}

\break

\addcontentsline{toc}{section}{References}
\bibliographystyle{plainnat}
\bibliography{paper}

\break

\addcontentsline{toc}{section}{Supporting tables and figures}
\section*{Supporting tables and figures}

\begin{table}[ht]
\caption{
    \label{tab:single_origin}
    \textbf{Best-fit parameters from the Single-Origin model.}
    \aprcomment{fill in caption}
}
\centering
\begin{tabular}[t]{lp{8cm}ll}
    \toprule
    Parameter & Description & Value & Std. err.\\
    \midrule
    $N_e$ & Ancestral effective population size & 10198 & 403 \\
    $N_{MH}$ & Size of modern-human lineage between Neanderthal and Nama splits & 21111 & 529 \\
    $N_{Nama_0}$ & Initial Nama size & 10224 & 370 \\
    $N_{Nama_F}$ & Final Nama size & 222 & 9 \\
    $N_{MSL_0}$ & Initial Mende size & 17211 & 769 \\
    $N_{MSL_F}$ & Final Mende size & 16822 & 606 \\
    $N_{EA}$ & Size of East African Branch & 7139 & 273 \\
    $N_{Gumuz_F}$ & Final Gumuz size & 3831 & 131 \\
    $N_{EP}$ & East African agriculturist size & 13033 & 491 \\
    $N_{GBR_0}$ & Initial British size & 846 & 33 \\
    $N_{GBR_F}$ & Final British size & 12121 & 507 \\
    $N_{Neand}$ & Neanderthal size & 1867 & 105 \\
    $T_{Nama}$ & Nama split time (years) & 110400 & 2525 \\
    $m_{Nama-MSL}$ & Nama-Mende symmetric migration rate & $2.82\times10^{-5}$ & $0.158\times10^{-5}$ \\
    $m_{Nama-EA}$ & Nama-East Africa symmetric migration rate & $4.94\times10^{-5}$ & $0.197\times10^{-5}$ \\
    \bottomrule
\end{tabular}
\end{table}

\end{document}
